leaper.py

```diff
@@ -135,7 +135,7 @@ def rdiff_link(self, src_project, src_package, src_rev, target_project, target_p
                 }
 
     def check_source_submission(self, src_project, src_package, src_rev, target_project, target_package):
-        self.logger.info("%s/%s@%s -> %s/%s"%(src_project, src_package, src_rev, target_project, target_package))
+        super(Leaper, self).check_source_submission(src_project, src_package, src_rev, target_project, target_package)
```

Comment:
just curious, is there an elegant way in python to avoid hardcoding the name of the class?